_episodes/2020-11-27-0246.md

```diff
@@ -10,3 +10,30 @@ summary: Discussing all things related to espionage
 
 - [Billion Dollar Spy](https://www.davidehoffman.com/)
   ([review](https://www.lawfareblog.com/true-cold-war-spy-thriller-contemporary-relevance))
+
+## From the Chat
+
+### Streaming Video Mentions
+
+| Program | Streaming service(s) |
+|---------|--------------------|
+| _The Americans_ | Amazon Prime |
+| _Babylon Berlin_ | Netflix |
+| _Berlin Station_ | US: EPIX trial on Prime Video<br/>Elsewhere: Netflix |
+| _Borgen_ | Netflix |
+| _The Good Shepherd_ | Hulu<br/>Amazon Prime Rental |
+| _Homeland_ | Amazon Prime |
+| _The Sopranos_ | HBO |
+| _The Wire_ | HBO |
+
+### Authors and Books
+
+| Author | Title | Notes |
+|--------|-------|-------|
+| Jo Nesbø | | author of Norwegian crime novels |
+| John le Carré | _Tinker, Taylor, Soldier, Spy_<br/>_The Honourable Schoolboy_<br/>_Smiley's People_<br/>_The Constant Gardener_<br/>_The Naive and Sentimental Lover_ | (writes Mike Godwin, "For a long time, I read all of LeCarre _[sic],_ but if you just read TINKER, TAILOR, then HONOURABLE SCHOOLBOY, then SMILEY'S PEOPLE, you've got the core of his best work.") |
+| Ulysses S. Grant | _The Personal Memoirs of U. S. Grant_ | |
+| Roger Hall | _You're Stepping on My Cloak and Dagger_ | |
+| Richard Hofstadter | _The Paranoid Style in American Politics_ | Here's a discussion with Sean Wilentz, Arlie Hochschild, Eric Foner, and Ira Katznelson that provides an up-to-date assessment of Hofstadter's work: [The Paranoid Style in American Politics: Revisiting Richard Hofstadter in the Time of Trump](https://youtu.be/zRLdLVfresI)|
+| Volker Kutscher | | Author of the books on which the Netflix series _Babylon Berlin_ is based |
+| David E. Hoffmann | _The Billion-Dollar Spy_ | Ben's review here: https://www.lawfareblog.com/true-cold-war-spy-thriller-contemporary-relevance |
```

Comment:
This is a duplicate of the existing entry above. I'd be fine with removing the above in favour of being consistent.

Reply:
No problem, that's fine with me. Do you want me to do it, or will you take care of it?

Reply:
I'll take care of it.

Reply:
Thanks!